diff -up gpsd-2.94/setup.py.silentmake gpsd-2.94/setup.py
--- gpsd-2.94/setup.py.silentmake	2010-04-20 11:37:21.000000000 +0200
+++ gpsd-2.94/setup.py	2010-04-30 14:48:19.000000000 +0200
@@ -56,7 +56,7 @@ if not 'clean' in sys.argv:
 gpspacket_sources = ["gpspacket.c", "packet.c", "isgps.c",
             "driver_rtcm2.c", "strl.c", "hex.c", "crc24q.c"]
 include_dirs = [ os.path.realpath(os.path.dirname(__file__)) ]
-version_out = os.popen(MAKE + " version")
+version_out = os.popen(MAKE + " --no-print-directory version")
 version = version_out.read()
 print(version)
 if version_out.close():
