diff -up gpsd-3.8/SConstruct.nolibcap gpsd-3.8/SConstruct
--- gpsd-3.8/SConstruct.nolibcap	2013-02-26 05:09:38.000000000 +0100
+++ gpsd-3.8/SConstruct	2013-02-27 11:09:11.561350794 +0100
@@ -490,7 +490,7 @@ else:
     confdefs.append("/* #undef HAVE_LIBRT */\n")
     rtlibs = []
 
-if config.CheckLib('libcap'):
+if False and config.CheckLib('libcap'):
     confdefs.append("#define HAVE_LIBCAP 1\n")
     # System library - no special flags
     caplibs = ["-lcap"]
